shared.css

```diff
@@ -0,0 +1,90 @@
+/* header styling */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem 2rem;
+}
+
+.header ul {
+  list-style: none;
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+}
+
+/* brand logo styling */
+
+.brand {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+}
+.brand__logo {
+  font-size: 1.5rem;
+  padding-right: 1rem;
+  color: var(--primary-color-dark);
+  border-right: 2px solid var(--primary-color-dark);
+}
+
+.brand__text {
+  letter-spacing: 0.5rem;
+  color: var(--primary-color-dark);
+  font-weight: bold;
+  font-family: 'Quintessential', cursive;
+}
+/* theme toggler */
+
+.theme-toggle {
+  position: relative;
+}
+.theme-toggle__checkbox {
+  position: absolute;
+  top: 0;
+  right: 0;
+  opacity: 0;
+}
+
+.theme-toggle__label {
+  padding: 5px;
+  background-color: var(--black-color);
+  border-radius: 0.625rem;
+  display: flex;
```

Comment:
check with the values in rem can you px if less than 10px

Reply:
If I take value in px then it looks bad in the mobile view due to the same border width as in the desktop view. So I changed that in the rem.